pkg/cmd/config.go

```diff
@@ -90,6 +96,20 @@ func setConfigValue(key, value string) error {
 	return saveConfig()
 }
 
+func listAllConfig(out io.Writer) error {
+	config, err := unmarshalConfig()
+	if err != nil {
+		return nil
+	}
+
+	// Iterating over structs is hard and there are only two configuration
+	// directives at time of writing. Hard-code the output format for now, and
+	// invest in a more maintainable method in the future.
+	_, err = fmt.Fprintf(out, "user.name=%s\nuser.email=%s\n",
+		config.UserName, config.UserEmail)
+	return err
```

Comment:
If you wanted to go for the more maintainable future you could achieve it by adding a String() to Config e.g. 

func (c Config) String() string {
	return fmt.Sprintf("user.name=%s\nuser.email=%s\n", c.UserName, c.UserEmail)
}

then the above becomes something like

_, err = fmt.Fprintf(out, "%s", config)

Essentially one place to format the struct, one place to do the IO

Go with what ever you feel is right though?